core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -76,7 +76,9 @@ object ApiVersion {
     // Introduced DeleteGroupsRequest V0 via KIP-229, plus KIP-227 incremental fetch requests,
     // and KafkaStorageException for fetch requests.
     "1.1-IV0" -> KAFKA_1_1_IV0,
-    "1.1" -> KAFKA_1_1_IV0
+    "1.1" -> KAFKA_1_1_IV1,
```

Comment:
Should this be KAFKA_1_1_IV0 ?

Reply:
This was based on the comments in lines 31-36, which states "When we change the protocol a second time while developing 0.10.0, we will add a new config value "0.10.0-IV1" and a corresponding case object KAFKA_0_10_0-IV1. We will change the config value "0.10.0" to map to the latest internal version object KAFKA_0_10_0-IV1." Following the example, I set 1.1 to the latest internal version, KAFKA_1_1_IV1.